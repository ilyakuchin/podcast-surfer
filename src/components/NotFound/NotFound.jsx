import React from 'react';
import { Header } from 'semantic-ui-react';
import { Link } from 'react-router-dom';

export default function NotFound() {
  return (
    <div>
      <Header as='h1'>404 Not found</Header>
      <Header as='h3'>
        <Link to='/'>Home</Link>
      </Header>
    </div>
  );
}
