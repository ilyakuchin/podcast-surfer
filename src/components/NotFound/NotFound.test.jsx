import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './NotFound';

configure({ adapter: new Adapter() });

describe('Test NotFound component', () => {
  test('should render correctly', () => {
    expect(toJson(shallow(<NotFound />))).toMatchSnapshot();
  });
});
