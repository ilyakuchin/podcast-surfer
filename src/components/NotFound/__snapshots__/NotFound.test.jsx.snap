// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NotFound component should render correctly 1`] = `
<div>
  <Header
    huge={true}
  >
    404 Not found
  </Header>
  <div>
    <Link
      to="/"
    >
      Home
    </Link>
  </div>
</div>
`;
